apps/backend/src/modules/projects/project.entity.ts

```diff
@@ -1,6 +1,6 @@
 import { type Entity } from "~/libs/types/types.js";
 
-import { type ProjectCreateResponseDto } from "./libs/types/types.js";
+import { type ProjectResponseDto } from "./libs/types/types.js";
```

Comment:
let's use ProjectGetAllItemResponseDto here. It should be ProjectGetByIdResponseDto, but for now these types are the same, so let's use it
```suggestion
import { type ProjectGetAllItemResponseDto } from "./libs/types/types.js";
```